scripts/can_bundle.py

```diff
@@ -79,23 +79,25 @@ def main(argv):
 def fetch_bundle(source: str, bundle_uuid: str, bundle_version: str) -> Bundle:
     for catalog in config.catalogs:
         plugin = plugin_for(catalog)
-        sources = set(map(str, plugin.sources))
         try:
             parsed_source = plugin.resolve_source(source)
-        except Exception:
-            pass
+        except Exception as e:
+            logger.debug('Skipping catalog %r (incompatible source)', catalog)
+            if config.debug > 1:
```

Comment:
I don't get this. 1 means debug logging for the Azul code. 2 means debug logging for everything. Since this is Azul code, why not

```suggestion
            if log.isEnabledFor(logging.DEBUG):
```

like everywhere else?

Reply:
The log statement wasn't working as intended so I removed it.